libavcodec/vvc/cabac.c

```diff
@@ -904,7 +878,7 @@ static int inline vvc_get_cabac(CABACContext *c, VVCCabacState* base, const int
     c->low  <<= lps_mask;
 
     if (!(c->low & CABAC_MASK))
-        vvc_refill2(c);
```

Comment:
I remember this will introduce some build errors for the window

could you "git cherry-pick ..ffvvc/workflow" and force-push your branch?
So we can now see the windows result

Reply:
For sure, I also tried it on Windows and everything works

Reply:
Great! please send to maling list for review. 
thank you